front/js/shared/tracking.es6

```diff
@@ -78,6 +67,44 @@
       });
   }
 
+  // @todo #759:30m Move PublishedGATracker to a separate file.
+
+  class PublishedGATracker {
```

Comment:
provide doc plz why it's published, but not common `GATracker`

Reply:
publish is not the most good name for classes purpose. `LimitedGATracher` or `SafeGATracker` might be more good. Up to you

Reply:
I believe `LoadedGATracker` would be the best name